web-console/1-homepage/catalog.md

```diff
@@ -36,9 +33,32 @@ On smaller viewport sizes, the catalog tabs should stack. ![hover](img/OpenShift
 - For secondary levels of categorization, there should again be an `Other` category if there are items to display (not shown in mockup).
 - Clicking on a subcategory will open a scrollable panel that contains the catalog items.
 
+### Responsive State
+![mobile-1](img/OpenShift-Next-Mobile-V2-scrolledup.png)
+
+- On smaller viewport sizes, the catalog tabs should stack with arrows to the right.
```

Comment:
Yep.  That's correct, @serenamarie125.  http://getbootstrap.com/css/#grid-options